coalib/collecting/Collectors.py

```diff
@@ -214,7 +214,7 @@ def collect_all_bears_from_sections(sections, log_printer):
     """
     Collect all kinds of bears from bear directories given in the sections.
 
-    :param sections:    list of sections from which bears are to be collected
+    :param sections:    list of sections so bear_dirs are taken into account
```

Comment:
for that we should document how doc comments should look like first, that's long overdue anyway